courses/free-courses-en.md

```diff
@@ -327,7 +327,9 @@
 * [A Tour Of Go](https://tour.golang.org/welcome/1)
 * [Go Tutorial](http://www.tutorialspoint.com/go/index.htm)
 * [Golang \| Gin HTTP Framework](https://www.youtube.com/playlist?list=PL3eAkoh7fypr8zrkiygiY1e9osoqjoV9w) - Pragmatic Reviews
+* [Gophercises: Free Coding Exercises for Budding Gophers](https://gophercises.com) - Jon Calhoun
```

Comment:
must add access note

Reply:
@eshellman thank you for the feedback, I removed the *Learn Go with Tests* book since I found it submitted under the books section and updated the Gophercises description with the access note.